src/server/ApiRoutes/ApiRoutes.js

```diff
@@ -1,40 +1,26 @@
 import express from 'express';
-import axios from 'axios';
 
-const router = express.Router();
+import Search from './Search';
+import appConfig from '../../../appConfig';
 
-function fetchApiData(url) {
-  return axios.get(url);
-}
+const router = express.Router();
 
 function MainApp(req, res, next) {
-  // Example of how to use the fetchApiData function:
-  // const apiUrl = '';
-  // fetchApiData(url)
-  //   .then(data => {
-  //     // Do something with the data
-  //     res.locals.data = {
-  //       Store: data,
-  //     };
-  //
-  //     next();
-  //   })
-  //   .catch(error => {
-  //     // Do something with the error
-  //     res.locals.data = {
-  //       Store: {},
-  //     };
-  //
-  //     next();
-  //   });
-
   res.locals.data = {
```

Comment:
This won't be needed anymore. There will probably be a different way of sending data from the server to the client. Redux calls it "hydrating" but not sure what it involves yet.